esphome/onju-voice-microwakeword.yaml

```diff
@@ -18,7 +18,7 @@ esphome:
   project:
     name: tetele.onju_voice_satellite
     version: "${project_version}"
-  min_version: 2024.6.0
+  min_version: 2024.7.0
```

Comment:
Also do not forget to update the [README](https://github.com/tetele/onju-voice-satellite/blob/e8f276fc65e6f1098e5326ee01c09c16743a46af/README.md?plain=1#L32)
I forgot this in my PR and addendum to the PR 😅